.circleci/config.yml

```diff
@@ -24,6 +24,9 @@ jobs:
       - run:
           name: Build Geth
           command: go run build/ci.go install
+      - run:
```

Comment:
It looks like you removed this bit on master and that it's no longer needed.

Reply:
Right, this PR isn't up to date with master.  Github says there are no conflicts, so it should come out alright, but I will merge master into this branch before merging to master so we can verify CI is ok before the final merge.

Reply:
Confirmed that these lines were removed when master was merged in (due to the #1245 which has been merged to master in the meantime).

Reply:
Merged in master (including the 1.9.12 merge PR which I merged to master earlier today), and confirmed that these lines were indeed removed as part of that.